### PR TITLE
Changes to nav, consolidate Getting Started README

### PR DESCRIPTION
**Pull request by omerbensaadon**
Early analytics are showing a 50% drop off after the first page. Based on user interviews, I noticed developers always scan for the

```
% Code Blocks %
```

I think we should consolidate the first two pages to see if this impacts drop-off rate. 

Also, changed some names in the Nav!

### CODE REVIEW

**Comment by omerbensaadon** [comment]
/cherrypick release-0.24

Going to wait a week to get more data, but eventually would like to merge this to main site to see impact on metrics.

**Comment by julz** [comment]
to be fair, I suspect a lot of the early analytics will be folks just wanting to check out the new site, so maybe not great data points (change is fine with me tho)

**Comment by omerbensaadon** [comment]
@julz it's the data combined with a gut feeling, going to wait a bit to check back on the data to push this to the live site

File: config/nav.yml
Comment (julz) [inline config/nav.yml:7]: do you want to also delete this file?

**Comment by omerbensaadon** [comment]
More evidence: 

Googling "Knative Quickstart" Yields "Before you Begin," ideally this would just be a page called Knative Quickstart which should improve SEO. 

https://www.google.com/search?q=knative+quickstart&rlz=1C5CHFA_enUS914US914&oq=knative+quickstart&aqs=chrome..69i57j69i60l3j69i64l3.4842j0j7&sourceid=chrome&ie=UTF-8

File: config/nav.yml
Comment (julz) [reply config/nav.yml:7]: ^ waiting on reply before lgtm-ing

**Review by julz — APPROVED** [review]
/lgtm

**Pull request merged**
